recipes/nifty/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "1.0.8" %}
+
+package:
+  name: nifty
+  version: {{ version }}
+
+source:
+    url: https://github.com/DerThorsten/nifty/archive/v{{ version }}.tar.gz
+    sha256: 4a70eb106a926271cf86cae3e568c71b26ca40ecdfbfc337110f485410aa2876
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - ninja  # [win]
+    - make
+  host:
+    - python
+    - numpy
+    - xtensor >=0.21,<0.22
+    - xtensor-python >=0.24,<0.25
+    - vigra
+    - hdf5  # [not win]
+    - z5py >=2.0.8  # [not win]
+    # macos needs boost as rt dependency because of boost filesystem
+    - boost-cpp  # [osx]
+    # build dependencies needed for z5
```

Comment:
Should `z5` be added as a separate package in conda-forge so it can be a dependency instead of vendoring it in?

If there is a reason not to the license implications should be figured out.

Reply:
z5 has a package on conda forge already: https://github.com/conda-forge/z5py-feedstock, which also builds the associated python bindings.

It's a header only dependency, but we need some more dependencies to build with it (which are not exported as runtime dependencies with z5py, because it's not necessary for the python pcakage).

Maybe the formulation `build dependencies needed for z5` is a bit misleading here, it's more accurate to say `dependencies for build with WITH_Z5=ON`.

Reply:
Ah I missed that it was header only, sounds good. I'll try to check back and merge when the CI finishes, ping me if you notice before I do :)